### ISSUE TITLE
Arducam Incorrect CCSDS Seconds Jumps in v1.6.1

### ISSUE THREAD

**Issue by kevincbruce** [nasa/nos3#251]
Arducam CCSDS Seconds value jumps by 2^24 (16,777,216) fairly regularly, rapidly inflating the CCSDS seconds. Doesn't seem to effect the times calculated by COSMOS for sent and received time.

**Comment by jlucas9** [comment]
It appears that possibly in v1.6.1 the CCSDS seconds for Arducam is invalid. Debugging:
* [NOS3 v1.6.1](https://github.com/nasa/nos3/tree/v1_06_01)
  * [components/arducam](https://github.com/nasa-itc/arducam/tree/c595adbd8a447278743f31f6177acd0c201fcd48)
    * [gsw/ARDUCAM/cmd_tlm/ARDUCAM_TLM.txt](https://github.com/nasa-itc/arducam/blob/c595adbd8a447278743f31f6177acd0c201fcd48/gsw/ARDUCAM/cmd_tlm/ARDUCAM_TLM.txt)
      * Both CCSDS_SECONDS and CCSDS_SUBSECONDS appear to have a BIG_ENDIAN conversion, which could cause the issue
* [NOS3 v1.6.1](https://github.com/nasa/nos3/tree/v1_06_01)
  * [components/generic_fss](https://github.com/nasa-itc/generic_fss/tree/ea121e1d851afe6bbc42c3b3528cffa3370c87d5)
    * [gsw/GENERIC_FSS/cmd_tlm/GENERIC_FSS_TLM.txt](https://github.com/nasa-itc/generic_fss/blob/ea121e1d851afe6bbc42c3b3528cffa3370c87d5/gsw/GENERIC_FSS/cmd_tlm/GENERIC_FSS_TLM.txt)
      * Both CCSDS_SECONDS and CCSDS_SUBSECONDS do not have the BIG_ENDIAN conversion
* It was my understanding that CCSDS headers were always BIG_ENDIAN
  * [CCSDS 133.0-B-2 Space Packet Protocol, Blue Book June 2020](https://public.ccsds.org/Pubs/133x0b2e1.pdf)
    * Section 3.4.2.3.3 Note 1 - "...The format of the secondary header, if present, is managed and mission specific."
    * Well that doesn't help really, let's compare to the latest `dev` and see
* [nos3 dev, ae43d88](https://github.com/nasa/nos3/commit/ae43d88c6a48e6f139b9e64e5444f72d0472a2d3)
  * [components/arducam](https://github.com/nasa-itc/arducam/tree/bafb309d18cf649cf451929381fb7e7b4e0e0dbb)
    * [gsw/ARDUCAM/cmd_tlm/ARDUCAM_TLM.txt](https://github.com/nasa-itc/arducam/blob/bafb309d18cf649cf451929381fb7e7b4e0e0dbb/gsw/ARDUCAM/cmd_tlm/ARDUCAM_TLM.txt)
      * Still has BIG_ENDIAN designation
* [nos3 dev, ae43d88](https://github.com/nasa/nos3/commit/ae43d88c6a48e6f139b9e64e5444f72d0472a2d3)
  * [componnts/generic_fss](https://github.com/nasa-itc/generic_fss/tree/c9d7f7dd3ba1cbb33a1bfe653d3f1d1e354a4838)
    * [gsw/GENERIC_FSS/cmd_tlm/GENERIC_FSS_TLM.txt](https://github.com/nasa-itc/generic_fss/blob/c9d7f7dd3ba1cbb33a1bfe653d3f1d1e354a4838/gsw/GENERIC_FSS/cmd_tlm/GENERIC_FSS_TLM.txt)
      * Still does not have BIG_ENDIAN designation
* So something has to be wrong here, let's run the latest and compare
  * Arducam counting up as expected
  * Generic FSS a large number jumping around, endianness issue

So in summary in NOS3 v1.6.1 CCSDS Seconds / Subseconds was reported in little endian and now in the latest dev branch (nearly two releases improved) it is reported as big endian. I'll take the action to update the various components to correct this in the latest!

**Comment by jlucas9** [comment]
Merged to `dev`
